remote/RemoteImGui.h

```diff
@@ -29,6 +29,28 @@ namespace imgui {
 		bool	KeysDown[256];
 	};
 
+	// Keys set in KeysDown to represent special keys
+	enum class ImGuiKeyCode : size_t {
+		Control = 21,
+		Shift = 22
+	};
+
+	enum class RemoteMessageType : unsigned char {
+		// System events fired by the relay server
+		RelayRoomJoined = 0,
+		RelayRoomUpdate,
+
+		// ImGUI events owned by the client
```

Comment:
nit: add comment that these values cannot be modified without updating the Javascript for both relay and standalone server